src/peer/room.js

```diff
@@ -60,6 +62,30 @@ class Room extends EventEmitter {
     this._pcConfig = this._options.pcConfig;
   }
 
+  /**
+   * Check whether the size of data to send is over the limits or not.
+   * @param {object} data - The data to check.
+   */
+  isOverLimits(data) {
+    const isBin = hasBin([data]);
+    const packet = {
+      type: isBin ? parser.BINARY_EVENT : parser.EVENT,
+      data: [data],
+    };
+    const encoder = new parser.Encoder();
+    let dataLength;
+    encoder.encode(packet, encodedPackets => {
+      dataLength = isBin
+        ? encodedPackets[1].byteLength
+        : encodedPackets[0].length;
+    });
+    const limits = 20 * 10 ** 6; // 20 MB
```

Comment:
it is not 20MB. Because 1KB = 1024B.
`20 * 1024 * 1024`

Reply:
These constants number should be defined at `src/shared/config.js`.